src/Functions/FunctionsCodingUUID.cpp

```diff
@@ -128,10 +128,14 @@ UUIDSerializer::Variant parseVariant(const DB::ColumnsWithTypeAndName & argument
     if (arguments.size() < 2)
         return UUIDSerializer::Variant::Default;
 
-    const auto representation = static_cast<magic_enum::underlying_type_t<UUIDSerializer::Variant>>(arguments[1].column->getInt(0));
-    const auto as_enum = magic_enum::enum_cast<UUIDSerializer::Variant>(representation);
+    Int64 value = static_cast<UInt64>(UUIDSerializer::Variant::Default);
+    if (!arguments[1].column->empty())
```

Comment:
Maybe we should enforce that the column must be const. Looks like right now we are just taking the value of the first row (which caused the crash when empty), which will be confusing if somebody passes a full column as the variant argument